mne/beamformer/_rap_music.py

```diff
@@ -144,19 +163,35 @@ def _make_dipoles(times, poss, oris, sol, gof):
     gof : array, shape (n_times,)
         The goodness of fit of the dipoles.
         Shared between all dipoles.
+    nave : int
+        Number of averages.
+    aspect_kind : int
+        The kind of data.
+    first : int
+        First sample.
+    last : int
+        Last sample.
+    comment : str
+        The dipole comment.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level
 
     Returns
     -------
     dipoles : list
-        The list of Dipole instances.
+        The list of DipoleFixed instances.
     """
     oris = np.array(oris)
 
     dipoles = []
     for i_dip in range(poss.shape[0]):
-        i_pos = poss[i_dip][np.newaxis, :].repeat(len(times), axis=0)
-        i_ori = oris[i_dip][np.newaxis, :].repeat(len(times), axis=0)
-        dipoles.append(Dipole(times, i_pos, sol[i_dip], i_ori, gof))
+        info = deepcopy(info)
+        info['pos'] = poss[i_dip][np.newaxis, :].repeat(len(times), axis=0)
+        info['ori'] = oris[i_dip][np.newaxis, :].repeat(len(times), axis=0)
+        info['gof'] = gof
+        info['amplitude'] = sol[i_dip]
```

Comment:
these are not standard info keys and won't get saved to disk in fif files.
see how IO code works for DipoleFixed